services/mac_manager/src/main/java/com/futurewei/alcor/macmanager/entity/MacAddress.java

```diff
@@ -0,0 +1,96 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+
+package com.futurewei.alcor.macmanager.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+
+
+@Data
+public class MacAddress {
+
+    public static final String MAC_DELIMITER = "-";
+    public static final int NIC_LENGTH = 24;
```

Comment:
Do you mean "MAC_ADDRESS_LENGTH“?

Reply:
No, MAC_ADDRESS_LENGTH is fixed as 48 ( OUI: 24, NIC: 24) NIC_LENGTH is the bytes we to generate mac address. By default, this is 24, but it can be 12, 6, etc in future.

Reply:
okay make sense.